pegasus/sites.v3/hourofcode.com/views/how-to/steps/steps_public_officials.haml

```diff
@@ -1 +1,151 @@
+:ruby
+  steps_educators_index = 9
 
+  steps_educators_heading = [
+    hoc_s(:hoc_how_to_step_label_invite_elected_official),
+    hoc_s(:hoc_how_to_step_label_register_event),
+    hoc_s(:hoc_how_to_step_label_choose_activities),
+    hoc_s(:hoc_how_to_step_label_plan_your_event),
+    hoc_s(:hoc_how_to_step_label_coordinate_media),
+    hoc_s(:hoc_how_to_step_label_confirm_details),
+    hoc_s(:hoc_how_to_step_label_get_coding),
+    hoc_s(:hoc_how_to_step_label_after_event),
+    hoc_s(:hoc_how_to_step_label_beyond_hoc)
+  ]
+
+  steps_educators_desc = [
+    hoc_s(:hoc_how_to_step_public_officials_invite_desc_01),
+    hoc_s(:hoc_how_to_step_public_officials_register_desc_01),
+    hoc_s(:hoc_how_to_step_choose_activities_desc_01),
+    hoc_s(:hoc_how_to_step_public_officials_plan_desc_01),
+    hoc_s(:hoc_how_to_step_public_officials_media_desc_01),
+    hoc_s(:hoc_how_to_step_public_officials_confirm_desc_01),
+    hoc_s(:hoc_how_to_step_public_officials_get_coding_desc_01),
+    hoc_s(:hoc_how_to_step_events_after_desc_01),
+    hoc_s(:hoc_how_to_step_beyond_advocacy_desc_01)
+  ]
+
+%section.steps
+  .wrapper
+
+    = view :"how-to/steps/side-nav/side_nav_public_officials"
+
+    %div.steps-wrapper
+      - steps_educators_index.times do |index|
+        .step{id: "step-0#{index + 1}", class:"#{'divider-accent' if index === 0 || index === 6 || index === 8}"}
+          .step-counter
+            %span
+              = index + 1
+          .step-content
+            %h3.heading-xl
+              = steps_educators_heading[index]
+            %p
+              = steps_educators_desc[index]
+
+            -# Step 1 additional content
+            - if index === 0
+              %p.tag.day-of
+                =hoc_s(:before_hoc)
+              %p
+                =hoc_s(:hoc_how_to_step_public_officials_invite_desc_02)
+              %figure
+                %img{src: "/images/fit-800/hoc_govs.png", alt: ""}
+                - if @country === "us"
+                  %figcaption.no-margin-bottom
+                    =hoc_s(:hoc_how_to_step_public_officials_invite_desc_03)
```

Comment:
Hiding this caption for those outside of the US since it's not relevant for other countries.